README.md

```diff
@@ -45,11 +49,26 @@ environment variable to include the jar which includes all dependencies:
 ```
 
-## Known Issues/TODO list
-- Currently working on packaging, discovered issue when connecting to S3 using packaged jar
-- S3 File System Store is likely not ideal sceanrio, other options to be investigated
-- S3 access currently very inefficient
-- Odd issue with data being lost when decompressing bytes in jzarr, an ugly hack is currently in place
-- Identification of S3 location needs updating
-- Refactor code to remove duplication
-- Parse colours for labels
+## Reader specific options
+
+The OMEZarrReader has a number of reader specific options in version 0.4.0 which can be used to customise the reader behaviour. This options can be used in the same manner as the reader options for Bio-Formats outlined [here](https://bio-formats.readthedocs.io/en/latest/formats/options.html#usage). 
+
+The new default behaviour of the `omezarr.include_labels` option introduced in v0.4.0 represents a change in behaviour from the v0.3 releases. Previously any Zarr arrays found in the labels folder would by default be represented as an additional image series. With the current default settings, Zarr arrays in the labels folder will no longer be included in the list of image series. Changing this setting to `true` will revert to the previous behaviour.
+
+In version v0.5.0 a new option `omezarr.alt_store` was added. This allows for the source of an alternative file store to be configured. Setting the option means the pixel data to be read from a different source than originally used when initialising the reader. The initial implementation was intended for use with the [IDR] (https://idr.openmicroscopy.org/), allowing IDR to read data directly from an S3 location. The current implementation only allows for S3 access when using a public https endpoint with unauthenticated access using anonymous credentials. A more complete, general purpose implementation will follow in a future release.
+
+An example of how this could be used would be to download locally a public dataset such as [6001240.zarr] (https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0062A/6001240.zarr). Setting the option as below, either via the API or using a `bfoptions` file, will allow you to call setID on the local file but have the pixel data read from the public S3 endpoint.
+ 
+```
+omezarr.alt_store = https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0062A/6001240.zarr
+```
+
+The list of available options are below:
+
+| Option | Default | Description |
+| --- | --- | --- |
+| `omezarr.quick_read` | false | Improves the read performance by limiting the number of files that are parsed. This assumes that the shape and resolution count of all images in a plate remains constant  |
+| `omezarr.save_annotations` | false | Determines if all the Zarr JSON metadata should be stored as XML annotations in the OME Model |
+| `omezarr.list_pixels` | false | Used to decide if getUsedFiles should list all of the pixel chunks |
```

Comment:
I thought the default value for `list_pixels` was `true` since that's what you get when you import a zarr (with no bfoptions). Then we set it to `false` with `bfoptions` for IDR?

Reply:
Yeah thats correct, good catch, updated in the latest commits.